content/docs/command-reference/add.md

```diff
@@ -336,29 +338,22 @@ $ tree .dvc/cache
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
 
-## Example: Transfer to the cache
+## Example: Transfer to an external cache
 
-When you have a large dataset in an external location, you may want to add it to
-the <abbr>project</abbr> without having to copy it into the workspace. Maybe
-your local disk doesn't have enough space, but you have setup an
-[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
-that could handle it.
+Sometimes you may want to add a large dataset currently found in an external
+location, so it becomes local to the project. However, your local file system
+may not have enough space to download it — which is needed to add data in DVC,
+right? Not necessarily!
```

Comment:
not sure about `Not necessarily!` ... Not necessarily _what_?

Reply:
To download it it (the large dataset) is not "necessarily needed" to add data in DVC — using the exact language from the p above.

But anyway, I rephrased this in simpler terms 👍